rpcd/playbooks/verify-maas.yml

```diff
@@ -0,0 +1,76 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Verify MaaS setup
+  hosts: hosts
+  user: root
+  pre_tasks:
+    # Raxmon client doesn't make it very easy to just get the full check or
+    # alarm names due to limited choices in formatting of output
+    - name: Enumerate all checks found
+      script: "{{ maas_rpc_scripts_dir }}/rpc-maas-tool.py --tab True --prefix {{ inventory_hostname }} checks"
+      register: maas_after_checks
+      changed_when: False
+      failed_when: maas_after_checks.rc != 0
+      tags:
+        - maas-verify
+    - name: Enumerate expected checks
+      shell: ls -1 /etc/rackspace-monitoring-agent.conf.d/ | sed -e 's/\.yaml$//'
+      when:
+        - maas_after_checks is defined
+      register: expected_checks
+      changed_when: False
+      tags:
+        - maas-verify
+    - name: Enumerate all alarms found
+      script: "{{ maas_rpc_scripts_dir }}/rpc-maas-tool.py --tab True --prefix {{ inventory_hostname }} alarms"
+      when:
+        - expected_checks is defined
+      register: maas_after_alarms
+      changed_when: False
+      failed_when: maas_after_alarms.rc != 0
+      tags:
+        - maas-verify
+  tasks:
+    - name: Verify presence of expected checks
+      fail: msg="Expected check ({{ item }}) not found"
+      with_items:
+        - "{{ expected_checks.stdout_lines }}"
+      when:
+        - maas_after_checks is defined
+        - maas_after_checks.stdout.find("{{ item }}") == -1
+      tags:
+        - maas-verify
+    - name: Verify absence of excluded checks
+      fail: msg="Excluded check ({{ item }}) found"
+      with_items:
+        - "{{ maas_excluded_checks }}"
+      when:
+        - maas_after_checks is defined
+        - maas_after_checks.stdout.find("{{ item }}") != -1
+      tags:
+        - maas-verify
+    - name: Verify presence of expected alarms
+      fail: msg="Expected alarm for check ({{ item }}) not found"
+      with_items:
+        - "{{ expected_checks.stdout_lines }}"
+      when:
+        - expected_checks is defined
+        - maas_after_alarms is defined
+        - maas_after_alarms.stdout.find("{{ item }}") == -1
+      tags:
+        - maas-verify
+  vars_files:
+    - "{{ rpc_repo_path }}/playbooks/roles/rpc_maas/defaults/main.yml"
```

Comment:
rpcd/playbooks/group_vars/all.yml:rpc_repo_path: /opt/rpc-openstack/openstack-ansible

That's the default so I think that would never work, since the role exists in /opt/rpc-openstack/rpcd